.github/workflows/node.js.yml

```
@@ -96,16 +96,25 @@ jobs:
           node scripts/ci/github-packages.mjs publish-cli "${{ steps.publish_core.outputs.core_version }}"
           echo "cli_version=$(node scripts/ci/github-packages.mjs ci-version forge-sql-orm-cli)" >> "$GITHUB_OUTPUT"
       - name: Merge library + EXTRA + CLI coverage into a single report
+        if: secrets.SONARCLOUD_TOKEN != '' || secrets.QLTY_COVERAGE_TOKEN != '' || secrets.CODACY_API_TOKEN != ''
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)